GameData/Kerbalism/Profiles/Default.cfg

```diff
@@ -214,19 +214,19 @@ Profile
     name = scrubber
     modifier = _Scrubber
     input = ElectricCharge@0.025
-    input = WasteAtmosphere@0.000020201
-    output = CarbonDioxide@0.020200830
+    input = WasteAtmosphere@0.020201
+    output = CarbonDioxide@0.020201
     dump = true
   }
 
   // convention: 1 capacity = enough to compensate for leaks in 70 m^2 surface (a cube of 40 m^3 volume)
   Process
   {
     name = pressure control
-    modifier = _PressureControl
+    modifier = _PressureControl,breathable
```

Comment:
what is the reason for the breathable modifier? (maybe add a comment?)

Reply:
ask yourself what the breathable modifier does and you will have your answer. If you do not know then you will have to learn by reading the code, same as everyone else

Reply:
I will make it easy for you this one time, in code the breathable modifier is true if the body a vessel is on has an atmosphere that is breathable, it is used by the rules and processes to determine if they should execute.
Having a breathable modifier attached to a rule or process in a .cfg file basically instructs that rule or process to stop running if it is in a breathable atmosphere.